src/routes/imgurRoutes.js

```diff
@@ -0,0 +1,73 @@
+const userRoutes = require('./userRoutes');
+const express = require('express');
+const fileUploadMiddleware = require('express-fileupload');
+const { ImgurClient } = require('imgur');
+const User = require('../models/User');
+
+const router = express.Router();
```

Comment:
Here you define a router. But in this file you're not adding any routes to this variable (instead you're defining `.post` on the `userRoutes` router). And then you export this `router` and tell Express to use it for anything starting with `/api/v1/profile` (in `expressServer.js`). But since you've not added any routes to this `router` variable, that means that there's nothing attached to `/api/v1/profile`

Reply:
![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/4d704b28-f074-4400-8e31-43a4c9e2c73d)

Reply:
Akos , I have made some modifications but I am not sure if it correct. Could you check please?